### PR TITLE
PMM-7705 technical preview features support

### PR DESCRIPTION
**Pull request by lunaticusgreen**
**What this PR does / why we need it**:  To give user better way to understand what features are technical preview.

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-7705

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by lunaticusgreen** [comment]
image
image

File: public/app/percona/settings/Settings.messages.ts [hunk 1]
```diff
@@ -25,6 +25,11 @@ export const Messages = {
     alertingLabel: 'Integrated Alerting',
     alertingTooltip: 'Option to enable/disable Integrated Alerting features.',
     alertingLink: `${HOW_TO_DOC_URL}/configure.html#integrated-alerting`,
+    technicalPreviewLegend: 'Technical preview features',
+    technicalPreviewDescription:
+      'These are technical preview features, not recommended to be used in productions environments. Read more\n' +
```
Comment (tiagomotasantos) [inline public/app/percona/settings/Settings.messages.ts:30]:
```suggestion
      'These are technical preview features, not recommended to be used in production environments. Read more\n' +
```

File: public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { getStyles } from './TechnicalPreview.styles';
+
+const ReadMoreLink = () => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <span>
+      {Messages.tooltipDescription}{' '}
+      <a href="https://per.co.na/pmm-feature-status" target="_blank" className={styles.link}>
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.tsx:13]: Move this link to constants file

File: public/app/percona/settings/components/Advanced/Advanced.styles.ts [hunk 3]
```diff
@@ -67,5 +67,41 @@ export const getStyles = stylesFactory(({ breakpoints, spacing, typography }: Gr
...
+        width: auto;
+        padding: 0 ${spacing.sm};
+      }
+    `,
+    technicalPreviewDoc: css`
+      display: flex;
+      align-items: center;
+      border: ${border.width.sm} solid ${colors.pageHeaderBorder};
+      border-radius: ${border.radius.sm};
+      color: #828282;
```
Comment (tiagomotasantos) [inline public/app/percona/settings/components/Advanced/Advanced.styles.ts:91]:
```suggestion
      color: ${colors.textWeak};
```

File: public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+export const getStyles = stylesFactory(
+  ({ breakpoints, spacing, typography, border, colors, palette }: GrafanaTheme) => {
+    return {
+      labelWrapper: css`
+        position: absolute;
+        top: 24px;
+        right: 24px;
+
+        h1 {
+          font-size: 14px;
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts:14]:
```suggestion
          font-size:${typography.size.md};
```

File: public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+          color: #828282;
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts:15]:
```suggestion
          color: ${colors.textWeak};
```

File: public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+          padding: 5px 8px 5px 8px;
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts:16]:
```suggestion
          padding: 5px ${spacing.sm} 5px ${spacing.sm};
```

File: public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+          border: 1px solid #9fa7b3;
```
Comment (tiagomotasantos) [inline public/app/percona/shared/components/Elements/TechnicalPreview/TechnicalPreview.styles.ts:17]:
```suggestion
          ${border.width.sm} solid ${colors.pageHeaderBorder};
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]
Beautiful!!!

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**